sci-libs/kissfft/kissfft-131.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="A Fast Fourier Transform (FFT) library that tries to Keep it Simple, Stupid"
+HOMEPAGE="https://github.com/mborgerding/kissfft"
+# QA checks don't like HOMEPAGE to be used in SRC_URI it seems
+BASE_URI="https://github.com/mborgerding/kissfft"
+
+LICENSE="BSD"
+IUSE="test"
+SLOT="0"
+RESTRICT="!test? ( test )"
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="${BASE_URI}"
+	inherit git-r3
+else
+	SRC_URI="${BASE_URI}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DEPEND=""
+RDEPEND=""
+
+src_install() {
+	dolib.so libkissfft.so
+	doheader kiss_fft.h
+}
+
+src_test() {
+	make -C test testcpp && test/testcpp || die
```

Comment:
Why not `emake`?

Reply:
You can build the tests in `src_compile` (`use test && make ...`) and run them here

Reply:
If I use emake I get a QA warning, I don't know if it's because it calls make inside the Makefile:
```
 * QA Notice: make jobserver unavailable:
 *
 * 	make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
```

Reply:
> You can build the tests in `src_compile` (`use test && emake ...`) and run them here

Some tests instead of compiling a test suite executes a python script, but before that it calls make to build some helper tools. It can be patched to separate those things but I don't know if it's worth the effort

Reply:
this is likely because the `Makefile` is broken